examples/grpc-plugin/README.md

```diff
@@ -0,0 +1,16 @@
+# grpc-plugin
+
+To run the `grpc-server` example, simply type in grpc-server folder:
+```
+go run main.go deps.go
```

Comment:
If there is a configuration file for gprc, please mention it here like we do for other examples, e.g.:
go run main.go deps.go [--grpc-config=<config-filepath>]

and add an example config grpc.conf with default values into the directory alongside the example.